src/components/cooperation-section-view/CooperationSectionView.tsx

```diff
@@ -36,15 +36,15 @@ const CooperationSectionView: FC<CooperationSectionViewProps> = ({
   )
 
   return (
-    <Box sx={styles.root}>
+    <Box key={id} sx={styles.root}>
```

Comment:
Why do we need this `key` prop here?

Reply:
Fixed in PR #2392. [CooperationSectionView.tsx](https://github.com/ita-social-projects/SpaceToStudy-Client/blob/e2311d4d43a0a766c187c3a8ea102bb58a06999b/src/components/cooperation-section-view/CooperationSectionView.tsx)